hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/troubleshooting_s3a.md

```diff
@@ -1155,6 +1155,25 @@ java.io.FileNotFoundException: Bucket stevel45r56666 does not exist
 Check the URI. If using a third-party store, verify that you've configured
 the client to talk to the specific server in `fs.s3a.endpoint`.
 
+#### <a name="preconditions_unmet"></a> `RemoteFileChangedException`: `Constraints of request were unsatisfiable`
```

Comment:
The http response is Precondition Failed, not "unmet". Even in AWS code, so I don't know where unmet keeps popping up.

https://github.com/aws/aws-sdk-java/blob/7bf0fbec42de8a7d54875510f27fb363ca2d19f5/aws-java-sdk-iot1clickdevices/src/main/java/com/amazonaws/services/iot1clickdevices/model/PreconditionFailedException.java#L37